plugins/woocommerce/includes/emails/class-wc-email-customer-pos-completed-order.php

```
@@ -324,17 +324,18 @@ public function email_footer( $email ) {
-		 * @param int      $item_id       Order item ID.
-		 * @param array    $item          Order item data.
-		 * @param WC_Order $order         Order object.
+		 * @param string                $qty_display The default quantity display.
+		 * @param WC_Order_Item_Product $item        The order item.
+		 * @param WC_Order              $order       The order object.
+		 * @return string The formatted quantity display with unit price.
 		 *
 		 * @internal For exclusive usage within this class, backwards compatibility not guaranteed.
 		 */
-		public function add_unit_price( $item_id, $item, $order ) {
+		public function format_quantity_with_unit_price( $qty_display, $item, $order ) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$qty_display'. (undefined)

(UnusedFormalParameter)